src/data/Department.java

```diff
@@ -0,0 +1,114 @@
+package data;
+
+import java.util.ArrayList;
+
+public class Department {
+    private String name;
+    private ArrayList<Employee> employees;
+    private int employeesNumber;
+    private double minimumSalary = Double.MAX_VALUE;
+    private double maximumSalary = Double.MIN_VALUE;
+    private double averageSalary;
+
+    public Department(String name, Employee employee) {
+        this.name = name;
+        this.employees = new ArrayList<>();
+        this.employees.add(employee);
+
+        calculateSalaryStatistics();
+    }
+
+    public Department(String name) {
+        this.name = name;
+        this.employees = new ArrayList<>();
+    }
+
+    public void addEmployee(Employee employee) {
+        employees.add(employee);
+        calculateSalaryStatistics();
+    }
+
+    private void calculateSalaryStatistics() {
+        if(employees.size()!=1) {
+            averageSalary=0;
+            for (Employee employee : employees) {
+                if (employee.getSalary() > maximumSalary) {
+                    maximumSalary = employee.getSalary();
+                } else if (employee.getSalary() < minimumSalary) {
+                    minimumSalary = employee.getSalary();
+                }
+                averageSalary += employee.getSalary();
+            }
+            employeesNumber = employees.size();
+            averageSalary = averageSalary / employees.size();
+
+        } else {
+            maximumSalary = employees.get(0).getSalary();
```

Comment:
to przerwie program dla pustej listy(a mozna stworzyc Departament z pusta llistą pracowników)

Reply:
Poprawiłe teraz przyjmuje w pliku wejściowym
;;;Security;